docs/src/forc/manifest_reference.md

```diff
@@ -0,0 +1,56 @@
+# Manifest Reference
+
+The `Forc.toml` (the *manifest* file) is a compulsory file for each package and it is written in [TOML] format. `Forc.toml` consists of the following fields:
+
+* [`[project]`](#the-project-section) — Defines a sway project.
+  * [`name`](#) — The name of the project.
+  * [`authors`](#) — The authors of the project.
+  * [`organization`](#) — The organization of the project.
+  * [`license`](#) — The project license.
+  * [`entry`](#) — The entry point of the project.
+  * [`implicit_std`](#) -  Controls whether provided `std` version (with current `forc` version) will get added as a dependency *implicitly*.  
```

Comment:
`_` for italics. Also, install mdlint for your IDE to auto-format and fix errors like extra or missing spacing.

```suggestion
  * [`implicit_std`](#) -  Controls whether provided `std` version (with current `forc` version) will get added as a dependency _implicitly_.
```